### ISSUE TITLE
Wallet Recovery

### ISSUE THREAD

**Issue by rachna-khatnawlia** [LtbLightning/bdk-rn#39]
How can we achieve Wallet Recovery using this Package?

**Comment by Czino** [comment]
Hi @rachna-khatnawlia, you can use the mnemonic parameter to instantiate the wallet

```
const result = await BdkRn.createWallet({
  mnemonic,
  password: '',
})
```

**Comment by BitcoinZavior** [comment]
Yes, `create` `wallet` with a known mnemonic is equivalent to wallet recovery.
Please note, the next version of the package will only allow wallet creation with descriptors
